index.html

```diff
@@ -1,270 +1,306 @@
+            </li>
+          </ul>
+        </li>
+        <li class="my-about-skill-container">
+          <div class="my-about-skill-heading">
+            <h4 class="my-about-heading-small">Frameworks</h4>
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr />
+        <li class="my-about-skill-container">
+          <div class="my-about-skill-heading">
+            <h4 class="my-about-heading-small">Skills</h4>
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr />
+     
+        </li>
+      </ul>
+          
       </div>
     </section>
-    <section>
-      <div class="contact-container">
-        <h3 class="contact-heading">Contact me</h3>
-        <p class="contact-paragraph">
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding, I'd love to help
-          with it.
-        </p>
-        <form class="contact-form" action="https://formspree.io/f/mjvdqdaj" method = "POST">
-          <input
-            class="contact-input"
-            required
-            maxlength="30"
-            name="name"
-            type="text"
-            placeholder="Please enter your name"
-          />
+    <!-- my Contact Me Section Starts -->
+    <section class="my-contact-me-form-container">
+        <h1 class="my-contact-me-form-heading">
+            Contact me
+        </h1>
+        <p class="my-contact-me-form-paragraph">
+            If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I'd love to help
+        with it
+      </p>
+      <form  action="https://formspree.io/f/mjvdqdaj"
+      method="POST" class="contact-me-form">
+        <input
+          class="contact-me-form-text-input"
+          required
+          maxlength="30"
+          name="name"
+          type="text"
+          placeholder="Your good name">
           <input
-            class="contact-input"
-            required
-            name="email"
-            type="email"
-            pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}"
-            placeholder="Your Email address"
-          />
+          class="contact-me-form-text-input"
+          pattern="[^ @]*@[^ @]*"
+          required
+          name="email"
+          type="email"
+          placeholder="Your email address"
+        />
           <textarea
-          class="contact-input"
+          class="contact-me-form-text-input"
           required
           maxlength="500"
           name="message"
           cols="30"
           rows="9"
-          placeholder="Write your message here."
+          placeholder="Write your message here"
         ></textarea>
-        <button class="contact-button" type="submit">
+         <button class="my-contact-me-form-button" type="submit">
```

Comment:
- [ ] As suggested by the previous reviewer, the **Get In Touch** button should be centrally aligned as in the Figma.
![btn](https://github.com/the-faizmohammad/my-portfolio/assets/99656687/89170cd1-2fb7-438f-966f-52ffcfae707c)